src/Controller/Admin/Document/DocumentController.php

```diff
@@ -522,15 +530,19 @@ public function updateAction(Request $request): JsonResponse
                         $this->updateIndexesOfDocumentSiblings($document, $request->get('index'));
                     }
 
-                    $success = true;
+                    $data = [
+                        'success' => true,
+                        'treeData' => $this->getTreeNodeConfig($document)
+                    ];
                     if ($oldPath && $oldPath != $document->getRealFullPath()) {
                         $this->firePostMoveEvent($document, $oldDocument, $oldPath);
                     }
                 } catch (Exception $e) {
                     return $this->adminJson(['success' => false, 'message' => $e->getMessage()]);
                 }
             } else {
-                $msg = 'Prevented moving document, because document with same path+key already exists or the document is locked. ID: ' . $document->getId();
+                $msg = 'prevented moving document, because document with same path+key already exists';
+                $msg .= ' aor the document is locked. ID: ' . $document->getId();
```

Comment:
```suggestion
                $msg = 'prevented moving document, because document with same path+key already exists' .
                    ' or the document is locked. ID: ' . $document->getId();
```